mentorship ios/Views/Home/Home.swift

```diff
@@ -14,6 +14,17 @@ struct Home: View {
         return homeViewModel.relationsListData
     }
     
+    var userFirstName: String {
+    
+        //Return just the first name
+        if let editFullName = self.homeViewModel.userName?.capitalized ?? "" {
```

Comment:
Line 20: Do we need nil coalescing here? Also, why are we capitalizing the name?

Reply:
Hi, thx for the review. So the self.homeViewModel.userName?.capitalized ?? "" was expressed that way in the project when I came to it so I used it that way in the computed var. Isn't userName an optional? I also left the chained capitalized as well because it was already there.